t/ui/24-feature-tour.t

```diff
@@ -58,7 +58,11 @@ $driver->find_element_by_link_text('Logout')->click();
 
 # quit tour temporarly
 $driver->get('/login?user=nobody');
-ok($driver->find_element('#step-0')->is_displayed(), 'tour popover is displayed');
+wait_until_element_shows_up(
+    selector                => '#step-0',
+    check_whether_displayed => 1,
```

Comment:
The name implies that the element is actually visible... it shows up, it doesn't just get loaded in the DOM.

I'd argue

    wait_until_element_shows_up '#step-0', 'tour popover is displayed'
    wait_for_element selector => '#step-0', is_displayed => 1
    wait_for_element selector => '#step-0', is_hidden => 1

And if we want a more generic option, it'd be nice to also use terminology from the web driver API.

Reply:
Makes sense. However, I'd prefer

```
wait_for_element selector => '#step-0', is_displayed => 0
```

instead of

```
wait_for_element selector => '#step-0', is_hidden => 1
```

because this way you can't combine these mutually exclusive options. Absence of the option would mean that there's no check at all (presence in the DOM tree is enough).